move2kube/exercise-3-customizations/exercise-3a-overriding-built-in-transformers/README.md

```diff
@@ -1 +1,130 @@
-# Under construction
+# Customizing generated Dockerfile and behavior of built-in transformer
+
+## Steps for Move2Kube CLI
+
+1. Run Move2Kube transform on the `enterprise-app` without any customizatoins and observe the output:
+
+    ```console
+    $ cd exercise-3-customizations/exercise-3a-overriding-built-in-transformers
+    ```
+
+    ```console
+    $ move2kube transform -s ../../source/enterprise-app/ --qa-skip --overwrite
+    ```
+
+2. List the files in the frontend service directory of the `enterprise-app`:
+
+    ```console
+    $ ls myproject/source/frontend
+    ```
+
+    As you notice, there are no scripts named `start-nodejs.sh` in the frontend service directory.
+    
+    ```console
+    Dockerfile        __mocks__         jest.config.js    package-lock.json src               test-setup.js     webpack.dev.js    LICENSE           dist              manifest.yml      package.json      stories           tsconfig.json     webpack.prod.js   README.md         dr-surge.js       nodemon.json      server.js         stylePaths.js     webpack.common.js
+    ```
+
+3. Check the generated Dockerfile for frontend service of the `enterprise-app`:
+
+    ```console
+    $ cat myproject/source/frontend/Dockerfile
+    ```
+
+    If we notice the Dockerfile generated for the frontend app, it uses registry.access.redhat.com/ubi8/nodejs-12 as base image.
+    
+    ```console
+    FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```

Comment:
Probably this is not the Dockerfile generated by latest version of Move2Kube. I think the base image used by Dockerfile in this case would be `registry.access.redhat.com/ubi8/nodejs-14`. Can you please check this once?
Corresponding to the nodejs version, we might need to also update the Dockerfile template of the custom transfomer, currently it says `quay.io/konveyor/nodejs-12`.

Reply:
Modified!